source/Mission.cpp

```diff
@@ -1659,3 +1668,10 @@ bool Mission::ParseContraband(const DataNode &node)
 
 	return true;
 }
+
+
+
+const bool Mission::SortHelper(const Mission &a, const Mission &b)
+{
+	return (a.order == b.order) ? a.name < b.name : a.order < b.order;
```

Comment:
Are these parentheses needed?
```suggestion
	return a.order == b.order ? a.name < b.name : a.order < b.order;
```
(don't apply the suggestion if you're not sure; I'm not a C++ expert)